data/bangs.json

```diff
@@ -30600,11 +30600,19 @@
   {
     "s": "Fallout Wikia",
     "d": "fallout.wikia.com",
-    "t": "fallout",
+    "t": "fallout-wikia",
     "u": "http://fallout.wikia.com/wiki/Special:Search?search={{{s}}}&fulltext=Search",
     "c": "Entertainment",
     "sc": "Games (specific)"
   },
+  {
+    "s": "Independent Fallout Wiki",
```

Comment:
I think this would be better as just "Fallout Wiki"

Reply:
Done :)